scripts/#6 Users with no orders.sql

```diff
@@ -0,0 +1,5 @@
+SELECT users.firstname, users.lastname, COUNT(orders.orderid) AS count
+FROM users LEFT JOIN orders
+ON users.userid = orders.orderid 

```

Comment:
`ON users.userid = orders.orderid` you want to join rows where the user id is equal order id but why? 
For example, you have two tables:
![image](https://user-images.githubusercontent.com/10303508/216162520-cc52d9d0-3c32-4cb0-90e8-7026081a5e86.png)
Try to answer what orders are Tomasz's and what orders are Michal's and maybe it helps you to fix this SQL script.